### PR TITLE
Merging Feature branch into master Branch

### PR DESCRIPTION
**Pull request by guillainbisimwa**
- Implemented Bubble sort
- Implemented Bubble sort by
- Used Ruby

### CODE REVIEW

File: bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      if array[i + 1] < array[i]
+        temp = array[i]
+        array[i] = array[i + 1]
+        array[i + 1] = temp
+      end
+      i += 1
+    end
+    j += 1
+  end
+  print array
```
Comment (kobiyoyo) [inline bubble_sort.rb]: remove `print` because ruby returns the last statement automatically

File: bubble_sort.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      if yield(array[i], array[i + 1]).to_i >= 0
+        swap = array[i]
+        array[i] = array[i + 1]
+        array[i + 1] = swap
+      end
+      i += 1
+    end
+    j += 1
+  end
+  print array
```
Comment (kobiyoyo) [inline bubble_sort.rb]: remove `print` because ruby returns the last statement automatically

File: bubble_sort.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+end
+
+bubble_sort([4, 3, 78, 2, 0, 2])
+
+puts "\n"
```
Comment (kobiyoyo) [inline bubble_sort.rb]: Remove this debugging code

File: bubble_sort.rb [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        array[i + 1] = swap
+      end
+      i += 1
+    end
+    j += 1
+  end
+  print array
+end
+
+bubble_sort([4, 3, 78, 2, 0, 2])
```
Comment (kobiyoyo) [inline bubble_sort.rb]: Remove this debugging code

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team    
#### Great Job so far, everything works fine, but there are a few more things required. 
### General 
- [x] How do I run this code if I wasn't familiar with programming ? That should be included in your readme.MD file
- [x] What is bubble sort ? That should be included also
- [x] For commits use imperative tense not past tense
- [x] I noticed commits just from one person, Lets not repeat this in feature projects
- [x] Check comments below
### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

**Review by expjazz — APPROVED** [review]
## Status: Project Approved :clap: 

Great work team :+1: 

You did all the changes and the algorithm is perfect.

Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Pull request merged**